tests/framework/extensions/workloads/pods/pod_status.go

```diff
@@ -40,28 +40,16 @@ func StatusPods(client *rancher.Client, clusterID string) ([]string, []error) {
 		podResults = append(podResults, "Pod's Status: \n")
 
 		for _, pod := range pods.Data {
-			podStatus := &corev1.PodStatus{}
-			err = v1.ConvertToK8sType(pod.Status, podStatus)
+			podResult, podError, err := CheckPodStatus(&pod)
 			if err != nil {
 				return false, err
```

Comment:
This should be set to nil, just in case pods aren't ready in the first go. We don't want to exit the poll too soon. That's why below we were return false, nil